components/compliance-service/api/reporting/server/server.go

```diff
@@ -337,6 +339,34 @@ func jsonExport(stream reporting.ReportingService_ExportServer) exportHandler {
 	}
 }
 
```

Comment:
Minor nit but we can actually refactor these export functions into one and pass in the type (json, XML or csv), since they are nearly identical.  The only difference between the three functions is what we do when it's not the initial run, and the conversion to XML or csv if not json.